### PR TITLE
Feature/consult overview association search

### PR DESCRIPTION
**Pull request by oscartison**
CLBV-64

### CODE REVIEW

File: app/controllers/index.js
Comment (claire-lovisa) [inline app/controllers/index.js:45]: I'm not sure this is actually useful or used ? It could be if you would want to display options for the `Filter verenigingen` field, but currently it's overridden by the model refresh in the route. Setting `this.model` like that in a controller is not really a good practice, it's the job of the route to define the model. The controller can update and save it when changed occur.

File: app/templates/association/representatives.hbs
Comment (claire-lovisa) [inline app/templates/association/representatives.hbs:3]: Same here, `showTableLoader` is not defined

**Review by claire-lovisa — CHANGES_REQUESTED** [review]
Aside from the in-code comments, I have a few questions/remarks:
- In the Jira task, there is a mention of English translation of some sentences but I don't see it appear in the PR, is it expected now or is it for later ?
- In the description of the PR, you put a direct link to the ticket, this should be avoided. We agreed at some point that we were allowed to write down the ticket number, but not to put the direct link :grin: 
- I see a lot of changes that are not directly related to this feature and some commits are shared between this branch and the branches of other PRs, which makes it harder to focus on the relevant parts while reviewing. For next times, can you make sure you only embedd your feature changes and keep it scoped to one branch ? Thanks ! For now, I'll wait until this PR is good to go and merged before reviewing the PRs 6 & 7, it should show less commits once most of the commits are in the dev branch.

**Comment by oscartison** [comment]
> Aside from the in-code comments, I have a few questions/remarks:
> 
>     * In the Jira task, there is a mention of English translation of some sentences but I don't see it appear in the PR, is it expected now or is it for later ?
> 
>     * In the description of the PR, you put a direct link to the ticket, this should be avoided. We agreed at some point that we were allowed to write down the ticket number, but not to put the direct link 😁
> 
>     * I see a lot of changes that are not directly related to this feature and some commits are shared between this branch and the branches of other PRs, which makes it harder to focus on the relevant parts while reviewing. For next times, can you make sure you only embedd your feature changes and keep it scoped to one branch ? Thanks ! For now, I'll wait until this PR is good to go and merged before reviewing the PRs 6 & 7, it should show less commits once most of the commits are in the dev branch.

@claire-lovisa Thank you for your time. For the English translations, I asked and the app does not need to be translated. It was only added for non-Dutch speaking developers. I'll make sure to keep your remarks in mind when making my next PR. 

I also tried to add loading states by taking inspiration from OP so I hope it's ok like this.

**Review by claire-lovisa — APPROVED** [review]
Thanks for the updates and the insights ! :)

**Pull request merged**
